### PR TITLE
Improve changelog readability

### PR DESCRIPTION
**Pull request by BasThomas**
Depends on https://github.com/Moya/contributors/pull/18

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
-- **Breaking Change** `Decodable` object mapping methods to `Moya.Response`. [#1335](https://github.com/Moya/Moya/pull/1335) by [@devxoul](https://github.com/devxoul).
-- **Breaking Change** `.requestJSONEncodable` case to `Task`. [#1349](https://github.com/Moya/Moya/pull/1349) by [@LeLuckyVint](https://github.com/LeLuckyVint), [@afonsograca](https://github.com/afonsograca) and [@sunshinejr](https://github.com/sunshinejr).
+- **Breaking Change** Added a `.parameterEncoding` case to `MoyaError`. [#1248](https://github.com/Moya/Moya/pull/1248) by [@SD10](https://github.com/SD10).
+- **Breaking Change** Added an `.objectMapping` case to `MoyaError`. [#1335](https://github.com/Moya/Moya/pull/1335) by [@devxoul](https://github.com/devxoul).
+- **Breaking Change** Added an`.encodableMapping` case to `MoyaError`. [#1349](https://github.com/Moya/Moya/pull/1349) by [@LeLuckyVint](https://github.com/LeLuckyVint), [@afonsograca](https://github.com/afonsograca) and [@sunshinejr](https://github.com/sunshinejr).
+- **Breaking Change** Added a `Decodable` object mapping methods to `Moya.Response`. [#1335](https://github.com/Moya/Moya/pull/1335) by [@devxoul](https://github.com/devxoul).
```
Comment (SD10) [inline Changelog.md]: My OCD is killing me -- technically this is not a breaking change as we've separated the breaking as its own entry. If you clear this up you'll be my hero 😅

Comment (BasThomas) [reply Changelog.md]: What do you mean? Is it or is it not a breaking change? Feel free to change!

Comment (SD10) [reply Changelog.md]: Yeah LN 8 is not a breaking change. The breaking component is listed on LN 6

**Comment by sunshinejr** [comment]
I've updated the [release notes](https://github.com/Moya/Moya/releases/tag/10.0.0-beta.1) as well 😉

**Pull request merged**